client/landing/gutenboarding/components/header/style.scss

```diff
@@ -51,11 +77,22 @@ $padding--gutenboarding__header: $grid-unit-10;
 }
 
 .gutenboarding__header-domain-picker-button {
-	padding: 0.3em 0.5em;
-}
+	max-width: 0;
+	opacity: 0;
+
+	&.has-content:not( .is-modal-open ):not( .is-open ):not( .picked-domain ) {
+		animation-duration: 5s;
```

Comment:
We could reduce the "disappearing effect" mentioned in https://github.com/Automattic/wp-calypso/pull/41788#issuecomment-624981030 by reducing this to `1s` or lower, but it does impact the animation effect.

The only other way I can think of is only apply the animation on the first suggestion, which might mean some extra JS.

Reply:
Would keeping all the animate-or-not logic in JS help, and apply some kind of `do-the-dance` CSS class to indicate that the component should animate?

Reply:
the 'disappearing effect' is just the animation being re-triggered because of the conditions added to animate the "wriggle" that happens when you chose a domain. 
It looks like it only happens when you close the dropdown without choosing a domain... This should be solvable, but it may need some study of how the different classes interact. 
It's a pretty minor case, I would say let's merge as it, and fix it afterwards (that or we could remove the 'wriggle' animation and keep the introduction one, and just add the other once we find how to keep both without side effects)